examples/tutorials/Introduction_to_PROTACs.ipynb

```diff
@@ -0,0 +1,1968 @@
+{
```

Comment:
<p>Worth converting this into Kd etc (use the gibbs formula).</p><p> </p><p>(from memory, apologize for errors)</p><p> </p><p>Delta G = RT ln K_d</p><p> </p><p>Can use to convert from Delta G to the concentration IC50 (roughly) for binding. (Simplifying the science)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4000/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>